Makefile

```diff
@@ -229,7 +229,6 @@ lint-full: $(GOLANGCI_LINT) ## Run slower linters to detect possible issues
 .PHONY: modules
 modules: ## Runs go mod to ensure proper vendoring.
 	go mod tidy
-	cd $(TOOLS_DIR); go mod tidy
 
```

Comment:
should we also remove al mentions to `TOOLS_DIR`?
```
TOOLS_DIR := hack/tools
TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/bin)
```

Reply:
not it is being used [here](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/Makefile#L51) and in other places in the makefile

Reply:
oh yeah, you are correct

Reply:
Please don't remove :)